### PR TITLE
Update lodestar libs

### LINKED ISSUE #156: Update lodestar libraries
Libraries to update to latest version:
- @chainsafe/bls
- @chainsafe/eth2.0-config
- @chainsafe/eth2.0-types
- @chainsafe/eth2.0-utils
- @chainsafe/ssz

### PR DESCRIPTION
**Pull request by morrigan**
Resolves #156 

Also replaced BN.js with BigInt

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ module.exports = {
         "node": true,
         "es6": true
     },
+    "globals": {
+        BigInt: true,
+    },
```
Comment (mpetrunic) [inline .eslintrc.js]: I think this will provide better typehinting:
```
    "lib": [
      "esnext.bigint"
    ],
```

File: package.json [hunk 2]
```diff
@@ -171,18 +171,18 @@
...
+    "@chainsafe/eth2.0-config": "^0.2.1",
+    "@chainsafe/eth2.0-types": "^0.2.1",
+    "@chainsafe/eth2.0-utils": "^0.1.1",
+    "@chainsafe/ssz": "^0.5.4",
     "@hapi/joi": "^16.1.7",
     "@nodefactory/bls-keystore": "^1.0.0-beta7",
     "axios": "0.19.0",
     "bech32": "1.1.3",
+    "bigint-buffer": "^1.1.5",
     "bip39": "3.0.2",
```
Comment (mpetrunic) [inline package.json]: I don't think we need this anymore as well as bech32

File: src/main/main.ts [hunk 3]
```diff
@@ -12,6 +14,7 @@ app.on("ready", async function() {
     await Promise.all([
         db.start(),
         createWindow(),
+        initBLS(),
```
Comment (mpetrunic) [inline src/main/main.ts]: Can you await initBLS before dba dn window start?

File: src/renderer/services/api/eth2.ts [hunk 4]
```diff
@@ -83,12 +81,12 @@ export class Eth2 implements IBeaconAPIClient {
         validatorPubkey: BLSPubkey,
         pocBit: uint8,
         slot: Slot,
-        shard: Shard
+        shard: number,
```
Comment (mpetrunic) [inline src/renderer/services/api/eth2.ts]: lel, leave this, I will correct this in Pr with validator api as this has become CommitteeIndex

File: test/unit/HDWallet.spec.ts [hunk 5]
```diff
@@ -7,6 +8,10 @@ const publicKeyStr =
     "0x95c259ecefa8e60edb099a6568887dd9ba35afa75062d01a12fcc5a40d402eca57afcdc1cff7e34fbcd84dfea0848bb9";
 
 describe(".generate()", () => {
+    beforeAll(async () => {
+        await initBLS();
+    });
```
Comment (mpetrunic) [inline test/unit/HDWallet.spec.ts]: Add this to setupFile:
https://jestjs.io/docs/en/configuration.html#setupfiles-array

File: .eslintrc.js (hunk 1 above)
Comment (morrigan) [reply .eslintrc.js]: I think you mean that for the tsconfig which I already did, this is for eslint

File: test/unit/HDWallet.spec.ts (hunk 5 above)
Comment (morrigan) [reply test/unit/HDWallet.spec.ts]: It doesn't work with async code, there are some hacks that don't look nice

File: src/renderer/models/account.ts [hunk 6]
```diff
@@ -98,6 +98,7 @@ export class CGAccount implements IAccount {
             try {
                 return await keystore.decrypt(password);
             } catch (e) {
+                console.error(e);
```
Comment (mpetrunic) [inline src/renderer/models/account.ts]: remove this :)

Comment (morrigan) [reply src/renderer/models/account.ts]: but I had a hard time catching a bug because of it :D

Comment (mpetrunic) [reply src/renderer/models/account.ts]: import and use electron-log :stuck_out_tongue:

Comment (morrigan) [reply src/renderer/models/account.ts]: ah, didn't notice that one 👼

**Pull request merged**